tdrs-frontend/nginx/local/default.conf.template

```diff
@@ -0,0 +1,144 @@
+        # We need to control cache headers for non-static files
+        # this is not applicable for frontend when Nginx is serving built files
+        proxy_cache tdp_cache;
+
+        ### ADD HEADERS
+        # for clickjacking
+        add_header X-Frame-Options SAMEORIGIN always;
+
+        # Using Adobe products like PDF, Flash, etc.? You can implement this header to instruct the browser
+        # how to handle the requests over a cross-domain. By implementing this header, you restrict loading
+        # your site’s assets from other domains to avoid resource abuse.
+        add_header X-Permitted-Cross-Domain-Policies: none;
+
+        # Referrer-Policy: no-referrer
+        add_header Referrer-Policy: no-referrer;
+
+        # Cross-Origin-Embedder-Policy:This response header (also named COEP) prevents a document from loading
+        # any cross-origin resources that don’t explicitly grant the document permission
+        add_header Cross-Origin-Embedder-Policy: require-corp;
+
+        # Isolates the browsing context exclusively to same-origin documents. Cross-origin documents are not loaded
+        # in the same browsing context
+        add_header Cross-Origin-Opener-Policy: same-origin;
+
+        # This response header (also named CORP) allows to define a policy that lets web sites and applications opt
+        # in to protection against certain requests from other origins (such as those issued with elements like
+        # <script> and <img>), to mitigate speculative side-channel attacks, like Spectre, as well as Cross-Site
+        # Script Inclusion (XSSI) attacks
+        add_header Cross-Origin-Resource-Policy: same-origin;
+
+        # The must-revalidate response directive indicates that the response can be stored in caches and can be
+        # reused while fresh. If the response becomes stale, it must be validated with the origin server before reuse.
+        add_header Cache-Control: max-age=604800;
+
+        # prevent MIME sniffing
+        add_header X-Content-Type-Options: nosniff;
+
+        # CSP header options. All options are set either to none or self except
+
+        set $CSP "default-src 'none';";
+        set $CSP "${CSP}script-src 'self';";
+        set $CSP "${CSP}style-src 'self' 'unsafe-inline' http://localhost:3000;";
+        set $CSP "${CSP}img-src 'self' data:;";
+        set $CSP "${CSP}font-src 'self';";
+        set $CSP "${CSP}connect-src 'self' localhost:*;";
+        set $CSP "${CSP}frame-src 'None';";
+        set $CSP "${CSP}manifest-src 'self';";
+        set $CSP "${CSP}object-src 'none';";
+        set $CSP "${CSP}frame-ancestors 'none';";
+        set $CSP "${CSP}child-src 'none';";
+        set $CSP "${CSP}media-src 'none';";
+        set $CSP "${CSP}prefetch-src 'none';";
+        set $CSP "${CSP}form-action 'none';";
+        set $CSP "${CSP}script-src-elem 'self' http://localhost:* http://www.w3.org;";
```

Comment:
```suggestion
        set $CSP "${CSP}script-src-elem 'self' http://localhost:*;";
```

Are we sure we want the w3 part in there? Confused on why. Are we pulling JS/CSS?

Reply:
I believe the icons needed w3. In any case, this should be safe.

Reply:
removed as suggested!